File: src/java/WebNinja/src/test/java/webninja/selenuim/IssueTable.java
package webninja.selenuim;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import webninja.Issue;

import java.util.ArrayList;
import java.util.List;

public class IssueTable {
    private final WebDriver webDriver;

    public IssueTable(WebDriver webDriver) {
        this.webDriver = webDriver;
    }


    public List<Issue> getIssues() {
        List<Issue> issues = new ArrayList<Issue>();
        List<WebElement> rows = webDriver.findElements(By.xpath("//table[@id='results']/tbody/tr"));
        for (WebElement row : rows) {
            String title = row.findElement(By.xpath("//td[4]")).getText();
            String severity = row.findElement(By.xpath("//td[3]")).getText();
            issues.add(new Issue(severity, title));
        }
        return issues;

    }
}

File: src/dotnet/WebNinja/WebNinja/selenium/IssueTable.cs
using System.Collections.Generic;
using OpenQA.Selenium;
using WebNinja.webninja;

namespace WebNinja.selenium
{
    public class IssueTable : PageObject
    {
        public IssueTable(IWebDriver driver) : base(driver)
        {
        }

        public IList<Issue> Issues
        {
            get
            {
                IList<Issue> issues = new List<Issue>();
                var rows = Driver.FindElements(By.XPath("//table[@id='results']/tbody/tr"));
                foreach (var row in rows)
                {
                    var title = row.FindElement(By.XPath("//td[4]")).Text;
                    var severity = row.FindElement(By.XPath("//td[3]")).Text;
                    issues.Add(new Issue(title,severity));
                }
                return issues;
            }
        }
    }
}